README.md

```diff
@@ -166,6 +166,7 @@ Table of Contents
   * [Apify](https://www.apify.com/) — Web scraping and automation platform that lets you create an API extracting websites data. Free tier with 10k monthly crawls and 7 days data retention.
   * [API Mocha](https://apimocha.com) - Completely free online API mocking for testing and prototyping.  Make up to 500 requests per day, fully customizable API responses, download mock rules as a Postman collection.
   * [Atlas toolkit](https://atlastk.org/) - Lightweight library to develop single-page web applications that are instantly accessible. Available for Java, Node.js, Perl, Python and Ruby.
+  * [Bhagavad Gita Api](https://vedicscripturesapi.herokuapp.com/) - A Completely free lightweight Node.js based [open source](https://github.com/vedicscriptures/bhagavad-gita-api) *Bhagavad Gita API*. 
```

Comment:
while this is nice and all, I dont think it really makes the cut for this list.  The list is primarily aimed at SaaS businesses with free tiers, the kind of thing you can do a development against and learn with and use in your own tools for free but later a client can pay them to provide a robust service.

As a tool thats just sitting on a heroku deployment - appears to even be the free one with the restrictions, slow starts etc - it just doesnt seem to be suitable for this list, 

thanks for the contribution though